languages/common-lisp/config.json

```
@@ -8,7 +8,14 @@
     "indent_size": 2
   },
   "exercises": {
-    "concept": [],
+    "concept": [
+      {
+        "slug": "basics",
+        "uuid": "6787f07b-6fca-48ec-9ed1-f37bb77126f2",
+        "concepts": ["comments", "expressions", "cons.basic", "symbols"],
```

In C# and F#, we're replaced these with a single concept (which is in the new [determining concepts document](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#basics)):

```suggestion
        "concepts": ["basics"],
```

This makes depending on them a bit easier, but feel free to use what works for your language.

Regarding `cons.basic`, that seems to imply a hierarchy, which we are steering people away from: see the [determining concepts document](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#basics).

---

Note that I've approved the PR. I just wanted to give the above two suggestions for consideration.

---

I think I might keep the concepts separate for the moment, but I'll keep that in mind if managing them gets tricky :)

As for `cons.basic`, I'll have an intermediate and advanced version. Is that still something I should steer away from? I remember someone mentioning that the dot notation could be used when there are several exercises on the same concept.

---

> I'll have an intermediate and advanced version. Is that still something I should steer away from?

Well, is it really the same concept or are there more advanced concepts introduced?

> I remember someone mentioning that the dot notation could be used when there are several exercises on the same concept

There was some confusion, which we hope to address with the section in the determining concepts document.

---

Well, half of me loves hierarchy, and the other half of me is one quite poor terms with it. I solve my moral dilemma, I've flattened quite a few concepts, so no more hierarchy!

I've changed things to just `cons`. Intermediate and advanced got broken up into small concepts :)

Thanks for pointing out those guidelines!